chapters/VK_KHR_swapchain/wsi.txt

```diff
@@ -258,7 +258,7 @@ flink:vkQueuePresentKHR any images it had already acquired from
 pname:oldSwapchain.
 E.g., an application may present an image from the old swapchain before an
 image from the new swapchain is ready to be presented.
-As usual, flink:vkQueuePresentKHR may: fail if pname:oldSwapchain has
+As usual, flink:vkQueuePresentKHR must: fail if pname:oldSwapchain has
```

Comment:
Let's drop the 'As usual,' clause, or link to a specific bit of the spec if there's a need to cite similar behavior.

Reply:
Dropped.